README.md

```diff
@@ -398,6 +398,11 @@ TripDetail tripDetail = amadeus.travel.tripParser.post(body);
 Destination[] directDestinations = amadeus.airport.directDestinations.get(Params
     .with("departureAirportCode","MAD")
     .and("max","2"));
+
+// Flight Availabilites Search POST
+// body can be a String version of your JSON or a JsonObject
+FlightAvailabilitySearch[] flightAvailabilitiesSearch
+  = amadeus.shopping.flightAvailabilitiesSearch.post(body);
```

Comment:
The path of the API is `/shopping/availability/flight-availabilities` so the methods to call should be `amadeus.shopping.availability.flightAvailabilities`